wpilib/wpilib/relay.py

```diff
@@ -57,21 +59,18 @@ class Value:
         #: Reverse
         kReverse = 3
 
-        _num_to_name = {
-            kOff: "Off",
-            kOn: "On",
-            kForward: "Forward",
-            kReverse: "Reverse",
-        }
-
-        _name_to_num = {
-            "Off": kOff,
-            "On": kOn,
-            "Forward": kForward,
-            "Reverse": kReverse,
-        }
-
-    class Direction:
+        @classmethod
+        def getPrettyValue(cls, num):
+            return cls(num).name[1:]
```

Comment:
I think you can make this a normal method instead, like upstream. I would personally prefer a property, but 🤷‍♂️.

Reply:
aha, I assumed get was returning ints